src/app/template.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="theme-color" content="#000000">
+        <meta name="description" content="Baseapp crypto-currency trading platform" />
+        <meta name="ui-version" content="%REACT_APP_GIT_SHA%"/>
+        <link rel="manifest" href="/manifest.json" async>
+        <link rel="shortcut icon" href="/favicon.ico" async>
+        <link rel="preload" href="https://fonts.googleapis.com/css?family=Roboto:400" as="font" type="font/woff2" crossorigin>
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,700" as="font" type="font/woff2" crossorigin>
+        <title>Cryptobase</title>
+    </head>
+    <body>
+        <script src="/config/env.js"></script>
```

Comment:
Don't you need "%PUBLIC_URL%" prefix? I'm not sure this will work in production, need to make a build and check.

Reply:
No it's don't need that prefix

Reply:
No, this is webpack's template html page. Relative path is enough.